torch/_inductor/triton_heuristics.py

```diff
@@ -490,9 +490,9 @@ def benchmark_all_configs(self, *args, **kwargs):
                     "%s: %f, nreg %d, nspill %d, #shared-mem %d",
                     k.config,
                     v,
-                    k.n_regs,
-                    k.n_spills,
-                    k.shared,
+                    k.n_regs if k.n_regs is not None else -1,
+                    k.n_spills if k.n_spills is not None else -1,
+                    k.shared if k.shared is not None else -1
```

Comment:
Looks like `k.shared` was moved to `k.metadata.shared`:
https://github.com/openai/triton/blob/9ff4870b2f386046054c6fe79c548ecdf9f2382d/python/triton/compiler/compiler.py#L315-L316

Reply:
Hi @peterbell10
Thank you! I will implement the changed property.